samza-log4j/src/main/java/org/apache/samza/logging/log4j/StreamAppender.java

```diff
@@ -68,6 +72,8 @@
   // Hidden config for now. Will move to appropriate Config class when ready to.
   private static final String CREATE_STREAM_ENABLED = "task.log4j.create.stream.enabled";
 
+  private static final String DROP_PRODUCER_ERRORS = "task.drop.producer.errors";
```

Comment:
Are you able to use the constant defined in `TaskConfig.DROP_PRODUCER_ERROR`?

Reply:
Yes.